lib/jsdom/browser/parser/xml.js

```diff
@@ -51,7 +51,22 @@ function appendChild(parent, child) {
 }
 
 function createParser(rootNode, ownerDocument, saxesOptions) {
-  const parser = new SaxesParser(saxesOptions);
+  const parser = new SaxesParser({
+    ...saxesOptions,
```

Comment:
ESLint is complaining about this. Could you modify https://github.com/jsdom/jsdom/blob/b4bb1c5248e05762aa3c4c14faa71ef3adf10657/.eslintrc.json#L8 to say 2018 (or 9) instead? We now require Node.js v8.x so using it should be fine.

Reply:
Done.